dockerfiles/jdk8/alpine/is/docker-entrypoint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# ------------------------------------------------------------------------
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
```

Comment:
Change the Licence Header

Reply:
fixed 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